composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "BSD-2-Clause",
     "type": "library",
     "require": {
-        "koala-framework/koala-framework": "4.0"
+        "koala-framework/koala-framework": "~4.1 || ~5.0"
```

Comment:
wenn dieser branch 4.1 und 5.0 unterstützt, braucht es nicht in 1.2 gehen sondern kann 1.1 sein, oder?

Reply:
Nein, `1.1` is nur für `KWF 4.0`, weil es da noch keinen Autoprefixer gibt.